boardswarm-cli/README.md

```diff
@@ -8,16 +8,16 @@ all options.
 The `auth` subcommand can be used to configure remote instances. To setup
 an instance while authenticating against an OIDC server one can simply run:
 ```
-$ boardswarm-cli  --instance <instance name> auth init  -u <instance url>
+$ boardswarm-cli  --instance <instance name> configure --new -u <instance url>
```

Comment:
eh? the auth command replace configure in the git version so this seems to be going backwards ;)